docs/decisions/0003-initial-api-specification.rst

```diff
@@ -0,0 +1,79 @@
+0003 Initial API Specification
+##############################
+
+Status
+******
+
+**Provisional**
+
+Context
+*******
+
+This doc describes the API specification for the ``subsidy-access-policy`` Django app in this repo.
+
+
+
+Open Questions and TODOs
+========================
+
+
+
+Primary Access Policy Route
+***************************
+**/api/v1/policy/**
+The root URL for all actions allowed on access policy apis.
+
+
+Enterprise Access Policy Transactions
+**************************************
+
+POST access policy redeem transaction
+======================================
+**/api/v1/policy/redeem/**
+
+Creats a new transaction to redeem an entitlement by making a request to ``POST /api/v1/subsidies/.../transactions/``
+A side-effect of a successful POST request here is the creation of a course enrollment or entitlement
+that "fulfills" the ledger transaction.
+
+Inputs
+------
+
+- ``group_id`` (POST data, required): The uuid of the customer. For now it will be an enterprise customer uuid.
```

Comment:
I think we can leave `group_id` out for now.  The customer uuid can be inferred from the `policy_uuid` in the route.